cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -33,6 +33,8 @@
 #include <cudf/utilities/traits.hpp>
 #include <cudf/utilities/type_dispatcher.hpp>
 
+#include <cuda/std/limits>
+#include <cuda/std/optional>
```

Comment:
Should we be using this over `thrust::optional` now?

Reply:
I believe so, similar to `cuda::std::pair`/`cuda::std::tuple` over `thrust::pair`/`thrust::tuple`.

Reply:
I think so. I believe it is strictly better now.